.zenodo.json

```diff
@@ -20,49 +20,50 @@
     {
       "orcid": "0000-0002-5094-3571",
       "affiliation": "Centre for Culture and Technology, Curtin University",
-      "name": "James P. Diprose"
+      "name": "Diprose, James P"
     },
     {
       "orcid": "0000-0001-8288-5241",
       "affiliation": "Centre for Culture and Technology, Curtin University",
-      "name": "Richard Hosking"
+      "name": "Hosking, Richard "
     },
     {
-      "name": "Richard Rigoni"
+      "name": "Rigoni, Richard"
     },
     {
       "orcid": "0000-0003-0485-3388",
-      "affiliation": "Curtin Institute for Data Science, Curtin University",
-      "name": "Aniek Roelofs"
+      "affiliation": "Centre for Culture and Technology, Curtin University",
```

Comment:
Wasn't Aniek part of the "Curtin Institute for Computation, Curtin University"?

Reply:
Thanks @jdddog - yes, I updated this due to the update of the Curtin Institute for Data Science name change. Aniek  does not have an ongoing affiliation with the CIDS through an adjunct position, but is named on the COKI website as COKI alumni.

Reply:
Thanks @kathrynnapier, I feel like it should be "Curtin Institute for Data Science" or "Curtin Institute for Computation" because the authorship is for work that was done when Aniek worked at the Curtin Institute for Computation, not for new work.

Reply:
Thanks @jdddog - please revert it to "Curtin Institute for Data Science".